taigaProject/src/taigaApi/controller/metric_controller.py

```diff
@@ -33,3 +34,8 @@ def get_cycle_time_metric(projectRequest: ProjectRequest, token: Annotated[str |
 @router.post("/metric/Burndown")
 def get_burndown_chart_metric(burndownChartRequest: BurndownChartRequest, token: Annotated[str | None, Header()] = None):
     return get_burndown_chart_metric_detail(burndownChartRequest.milestoneId, burndownChartRequest.attributeKey, token)
+
+@router.post("/metric/Cruft")
+def get_zero_business_value(cruftRequest: CruftRequest, token: Annotated[str | None, Header()] = None):
+    project_info = get_project_by_slug(cruftRequest.projectSlug, token)
```

Comment:
You can get the project id from UI along with the start and end dates. So you can remove this call from here.
Also, change **projectSlug** to **project_id** in the CruftRequest.

Reply:
Made the change